c/include/proton/annotations.h

```diff
@@ -90,4 +90,24 @@
     #endif
 #endif
 
+// fallthrough
+#if defined __cplusplus && defined __has_cpp_attribute
+    #if __has_cpp_attribute(fallthrough) && __cplusplus >= __has_cpp_attribute(fallthrough)
+        #define PN_FALLTHROUGH [[fallthrough]]
+    #endif
+#endif
+#if !defined PN_FALLTHROUGH && defined __STDC_VERSION__ && defined __has_c_attribute
```

Comment:
Could be `#elif`

Reply:
For this c/c++ check, yes, this is exclusive, but I can't replace all the #endif #if with #elif here.

I changed this one line.